docs/execute-scripts.md

```diff
@@ -45,9 +45,9 @@ The `Type` must be the same as type in the script source code for that argument.
 - Flag: `--args-json`
 - Valid inputs: arguments in JSON-Cadence form.
 
-Arguments passed to the Cadence script in `Type:Value` format.
-The `Type` must be the same type as the corresponding parameter
-in the Cadence source code.
+Arguments passed to the Cadence transaction in Cadence JSON format.
```

Comment:
This file is about scripts. Is `transaction` ok here?

Reply:
Damn, another copy-paste error :/